packages/gatsby-page-utils/src/ignore-path.ts

```diff
@@ -37,5 +37,5 @@ export function ignorePath(
       settings.patterns = ignore.patterns
     }
   }
-  return mm.any(path, settings.patterns, settings.options)
+  return isMatch(path, settings.patterns, settings.options)
```

Comment:
`any` is an undocumented alias for `isMatch` - this change calls the exact same method, now by its documented name